prismic-model/js/exhibitions.js

```diff
@@ -86,6 +86,11 @@ const Exhibitions = {
       }
     }
   },
+  Resources: {
```

Comment:
I'd check with the content team, but this might be worth sticking in the main section?

Reply:
Just ran this past Jen, she likes it like this